src/python_testing/conformance_support.py

```diff
@@ -250,66 +315,70 @@ def __call__(self, feature_map: uint, attribute_list: list[uint], all_command_li
             raise ConformanceException('NOT operation on optional or disallowed item')
         # Features in device types degrade to optional so a not operation here is still optional because we don't have any way to verify the features since they're not exposed anywhere
         elif decision == ConformanceDecision.OPTIONAL:
-            return ConformanceDecision.OPTIONAL
+            return decision
```

Comment:
comment about return types seems to apply throughout the PR.

Reply:
as above - changed this to be more explicit by checking against the data member directly and removing the equality operator.